core/txn/mod.go

```diff
@@ -15,6 +15,7 @@ type Transaction interface {
 	// GetID returns the unique identifier for the transaction.
 	GetID() []byte
 
+	// GetNonce returns the sequence number of the transaction.
```

Comment:
somehow confusing. We already have a nonce that is associated with an identity in the validation package. Maybe here it should be called `GetSeqNum()` ?

Reply:
Hm actually the comment is unclear, because it's the sequence number of the transaction *for a given identity*

Reply:
The validation package is using this one so they don't differ from each other.

Reply:
Oh right, makes sense then. I just got confused by the comment